packages/contracts/utils/testHelpers.js

```diff
@@ -940,16 +940,20 @@ class TestHelper {
 
   // --- Redemption functions ---
 
-  static async redeemCollateral(redeemer, contracts, LUSDAmount, maxFee = this._100pct) {
+  static async redeemCollateral(redeemer, contracts, LUSDAmount, gasPrice = 0, maxFee = this._100pct) {
```

Comment:
Changing arguments position could result in unexpected behavior, which actually happened and tests are failing now

@edmulraney  @RickGriff